src/Migrations/Version20190430070122.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+declare(strict_types=1);
+
+namespace DoctrineMigrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ * @codingStandardsIgnoreFile
+ */
+final class Version20190430070122 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf(
+            $this->connection->getDatabasePlatform()->getName() !== 'mysql',
+            'Migration can only be executed safely on \'mysql\'.'
+        );
+
+        $this->addSql('CREATE TABLE technology_user (technology_id INT NOT NULL, user_id INT NOT NULL, INDEX IDX_2B90AC434235D463 (technology_id), INDEX IDX_2B90AC43A76ED395 (user_id), PRIMARY KEY(technology_id, user_id)) DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci ENGINE = InnoDB');
+        $this->addSql('ALTER TABLE technology_user ADD CONSTRAINT FK_2B90AC434235D463 FOREIGN KEY (technology_id) REFERENCES technology (id) ON DELETE CASCADE');
+        $this->addSql('ALTER TABLE technology_user ADD CONSTRAINT FK_2B90AC43A76ED395 FOREIGN KEY (user_id) REFERENCES user (id) ON DELETE CASCADE');
+        $this->addSql('DROP TABLE activity');
```

Comment:
It seems when you first ran the migrations you had tables in database. Cleanup them:
- Remove the migrations.
- Remove everything from the database.
- Generate the migrations.
- Commit.

Reply:
Don't forget about migrations.